src/main/java/org/twenty092023/FindPrimeNumber.java

```diff
@@ -0,0 +1,16 @@
+package org.twenty092023;
+
+public class FindPrimeNumber {
+    public static void main(String args[]) {
+        for (int i = 100; i <= 200; i++) {
```

Comment:
Write a method isPrimeNumber that accept one integer method should return boolean value. And call this method inside main method, call this method inside for loop to find all prime numbers exist between 100 and 200.

and write junit to test API.

Reply:
junit tutorial: https://www.vogella.com/tutorials/JUnit/article.html